cmd/mattermost/commands/channel_test.go

```diff
@@ -113,3 +114,17 @@ func TestCreateChannel(t *testing.T) {
 	name = name + "-private"
 	CheckCommand(t, "channel", "create", "--display_name", name, "--team", th.BasicTeam.Name, "--private", "--name", name)
 }
+
+func TestRenameChannel(t *testing.T) {
+	th := api4.Setup().InitBasic()
+	defer th.TearDown()
+
+	channel := th.CreatePublicChannel()
+	CheckCommand(t, "channel", "rename", th.BasicTeam.Name+":"+channel.Name, "newchannelname10", "--display_name", "New Display Name")
+	RunCommand(t, "channel", "rename", th.BasicTeam.Name+":"+"channel.Name", "newchannelname10", "--display_name", "New Display Name")
```

Comment:
I think this second line (`RunCommand` one) is not necessary, actually, I doesn't check anything.

Reply:
You're right!